src/main.rs

```diff
@@ -142,6 +150,18 @@ enum Command {
     Edit {
         /// The citation key to edit.
         citation_key: RecordId,
+        /// Normalize whitespace
+        #[clap(long)]
+        normalize_whitespace: bool,
+        /// Set "eprint" and "eprinttype" BibTeX fields.
+        #[clap(long, value_delimiter = ',')]
+        set_eprint: Vec<String>,
+        /// Do not open the editor.
+        ///
+        /// This is mainly useful for the non-interactive edit methods, such as
+        /// `--normalize_whitespace` and `--set_eprint`.
+        #[clap(long)]
+        non_interactive: bool,
     },
```

Comment:
Just noticing the `#[clap(...)]`s here. I think they should be `#[arg(...)]`, at least according to the [official reference](https://docs.rs/clap/latest/clap/_derive/index.html). Not sure what different it makes.

Same for the `Find` command below.